sevenseconds/config/iam.py

```diff
@@ -36,6 +36,17 @@ def effective_roles(config):
     return roles
 
 
+def effective_attached_policies(config, role_name, role_cfg):
+    attached_policies = role_cfg.get("attached_policies", [])
+    additional_attached_policies = []
+    for additional_attached_policy in config.get("additional_attached_policies", []):
+        role = additional_attached_policy["role"]
+        if role == role_name:
+            policies = additional_attached_policy.get("policies", [])
+            additional_attached_policies += [policy for policy in policies if policy not in attached_policies]
```

Comment:
You could use a set here to avoid this kind of filtering. It'll be converted into a set anyway, so I'm not sure if duplicates matter at all.

Reply:
good idea